recipe/meta.yaml

```diff
@@ -61,7 +65,7 @@ outputs:
         - zlib
         - mpi 1.0 openmpi
       run_constrained:
-        - cudatoolkit >=9.2  # [linux64]
+        - cudatoolkit >={{ cuda_compiler_version }}  # [linux64]
```

Comment:
This was hardcoded as 9.2, and I don't understand the reasoning there. Maybe it should still be 9.2? Or not version-pinned at all?